doc/source/installing.rst

```diff
@@ -80,6 +80,23 @@ Or using the Anaconda/Miniconda Python distributions
   $ conda install --channel conda-forge yt
 
 
+Additional requirements (pip only)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+yt knows about several data file formats. In many cases (e.g. HDF5), additional
+dependencies are needed to enable parsing, that are not installed with yt by default.
+In order to install every requirements for a specific format alongside yt,
+one can specify them as, for instance
+
+.. code-block:: bash
+
+  $ python -m pip install --upgrade pip
+  $ python -m pip install --user "yt[ramses]"
+
+Extra requirements can be combined, separated by commas (say ``yt[ramses,enzo_e]``).
+Note that all format names are normalized to lower case.
```

Comment:
So format names are normalized, but `Fortran` and `HDF5` aren't, right?

Reply:
Right. I'm using the word "format" here to avoid too specific nomenclature with frontends and dataset classes. I don't really intend `HDF5` and `Fortan` target as user exposed (they are, of course, but I'm intentionally not advertising them in docs)